packages/aggregator/src/connections/EntityConnection.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { ApiConnection } from '@tupaia/server-boilerplate';
+
+const { ENTITY_SERVER_API_URL = 'http://localhost:8050/v1' } = process.env;
+
+const createDescendantsUrl = (hierarchyName, entityCodes, dataSourceEntityType) =>
+  `hierarchy/${hierarchyName}/descendants?entities=${entityCodes.join(
+    ',',
+  )}&field=code&descendant_filter=type:${dataSourceEntityType}`;
+
+const createRelationsUrl = (
+  hierarchyName,
+  entityCodes,
+  aggregationEntityType,
+  dataSourceEntityType,
+) =>
+  `hierarchy/${hierarchyName}/relations?entities=${entityCodes.join(
+    ',',
+  )}&field=code&groupBy=descendant&${
+    aggregationEntityType === 'requested' ? '' : `ancestor_filter=type:${aggregationEntityType}`
```

Comment:
This is a new supported aggregationEntityType, `requested`. It will aggregate dataSourceEntities to the requested entities. It is very useful but I feel it should be more clearly signified somewhere...